src/nupic/data/record_stream.py

```diff
@@ -378,68 +402,93 @@ def clearStats(self):
 
   @abstractmethod
   def getError(self):
-    """Returns errors saved in the storage."""
+    """:returns: errors saved in the storage."""
 
 
   @abstractmethod
   def setError(self, error):
-    """Saves specified error in the storage."""
+    """
+    Saves specified error in the storage.
+
+    :param error: Error to store.
+    """
 
 
   @abstractmethod
   def isCompleted(self):
-    """Returns True if all records are already in the storage or False
-    if more records is expected.
+    """
+    :returns: True if all records are already in the storage or False
+              if more records is expected.
     """
 
 
   @abstractmethod
   def setCompleted(self, completed):
-    """Marks the stream completed (True or False)."""
+    """
+    Marks the stream completed.
+
+    :param completed: (bool) is completed?
+    """
 
 
   @abstractmethod
   def getFieldNames(self):
-    """Returns an array of field names associated with the data."""
+    """
+    :returns: (list) of field names associated with the data.
+    """
 
 
   @abstractmethod
   def getFields(self):
-    """Returns a sequence of nupic.data.fieldmeta.FieldMetaInfo
-    name/type/special tuples for each field in the stream. Might be None, if
-    that information is provided externally (thru stream def, for example).
+    """
+    :returns: (list) of :class:`nupic.data.fieldmeta.FieldMetaInfo`objects for
+        each field in the stream. Might be None, if that information is provided
+        externally (thru stream def, for example).
```

Comment:
Space missing between :class:`nupic.data.fieldmeta.FieldMetaInfo`  and `objects`, so the formatting is off in the HTML.

Super minor: consider spelling `thru` -> `through`.